src/javascript/_common/base/gtm.js

```diff
@@ -12,13 +12,14 @@ const getPropertyValue = require('../utility').getPropertyValue;
 const getAppId         = require('../../config').getAppId;
 
 const GTM = (() => {
-    const isGtmApplicable = () => (/^(1|1098|14473|15284)$/.test(getAppId()));
+    const isGtmApplicable = () => (/^(1|1098|14473|15284|1108)$/.test(getAppId()));
```

Comment:
I should remove this before merging the PR

Reply:
please remember to remove this